Assignment-07-Product Designer/ui-training-assignment-7.htm

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
```

Comment:
Please ensure UX to UI 100% match

Reply:
ui to ux match with proper responsive media query